feature/experimental/gribi/ate_tests/route_removal_during_failover_test/README.md

```diff
@@ -0,0 +1,55 @@
+# TE-13.2: gRIBI route DELETE during Failover 
+
+## Summary
+
+Validate gRIBI route flush during SSO
+
+## Procedure
+
+*   Connect ATE port-1 to DUT port-1, and ATE port-2 to DUT port-2.
+
+*   Create 64 L3 sub-interfaces under DUT port-2 and corresponding 64 L3 sub-interfaces on ATE port-2.
+
+*   Establish `gRIBI client` connection with DUT negotiating `FIB_ACK` as the requested ack_type.
+
+*   Install 64 L3 sub-interfaces IP to NextHopGroup(NHGID: `1`) containing all of the nexthops specified to ATE port-2.
+
+*   Inject `1000` IPv4Entries(IPBlock1: `198.18.196.1/22`) in default VRF with NHGID: `1`.
+
+*   Validate that the entries are installed as FIB_PROGRAMMED using getRPC.
+
+*   Send traffic from ATE port-1 to prefixes in IPBlock1 and ensure traffic flows 100% and reaches ATE port-2.
+
+*   Start flushing  IPv4Entries((IPBlock1: `198.18.196.1/22`) in default VRF with NHGID: `1`. Concurrently, trigger a supervisor switchover using gNOI `SwitchControlProcessor`  while IPBlock1 entries are only partially flushed.
+
+*   Following reconnection of the `gRIBI client` to a new master supervisor, validate if partially deleted entries of IPBlock1  are not present in the FIB using a get RPC.
```

Comment:
I think practically it's very hard to guarantee that the switchover happens while the entries are partially flushed out (Or maybe some vendor's switchover is always blocked by the Flush request).

Instead, I think the focus of the test should be: 
* If you received `FlushResponse == OK`, but those entries are still exist after switchover, then that's an error. 
* If you don't receive `FlushResponse == OK`, those entries are ok to be still there.
* If there is any new coredumps, that's an error. 

What do you think?

Reply:
It does makes sense to add this verification. I think for the second case when FlushResponse is not received , we can verify for either some or none of the entries are present. This will take care of the corner case where all the prefixes were flushed but the response was not sent/received due to switchover.

Reply:
updated , please check.